proposals/lip-0042.md

```diff
@@ -78,6 +88,38 @@ This module does not define any state store.
 
 This module does not define any command.
 
+### Events
+
+#### RewardMinted
+
+Event is emitted when the reward is minted, it carries information about reward reduction. This event has `typeID = TYPE_ID_REWARD_MINTED`.
```

Comment:
```suggestion
The event is emitted when the reward is minted. In case of a reward reduction, it provides information about the reason for the reduction. This event has `typeID = TYPE_ID_REWARD_MINTED`.
```

Reply:
Changed as per suggestion.